include/geom/bounding_box.h

```diff
@@ -95,7 +95,7 @@ class BoundingBox : public std::pair<Point, Point>
   bool intersects (const BoundingBox &) const;
 
   bool intersect (const BoundingBox & b) const
-    { libmesh_deprecated(); return this->intersect(b); }
+    { libmesh_deprecated(); return this->intersects(b); }
```

Comment:
Ha!  I was just about to ask you if you would mind if I cherry-picked it over to master to fix it ASAP.